src/events/server/threadCreate.ts

```diff
@@ -21,9 +21,12 @@ const event: Event = {
 				lastMessage?.system &&
 				lastMessage.author.id === client.user!.id
 			) {
-				if ((record && record.SupportChannel === parentId) || name.toLowerCase() === 'support') {
-					lastMessage.delete();
+				if (!record || (record && !record.ThreadNotifications)) {
```

Comment:
I wasn't able to test this as the system thread notification message never popped up despite commenting out all of the code. Does that message appear for you, and if it does, does the code work as intended?